src/main/java/com/example/java_pro_homework_7/Pet/Animal.java

```diff
@@ -0,0 +1,70 @@
+package com.example.java_pro_homework_7.Pet;
+
+public abstract class Animal {
+
+    String name;
+    String color;
+    String breed;
+    int limitDistanceRun;
+    int limitDistanceSwim;
+    protected int distanceRun;
+    protected int distanceSwim;
+    private int animalCounter;
```

Comment:
`animalCounter` should be static, or otherwise, your counter won't work correctly.